system/Model.php

```diff
@@ -348,6 +348,42 @@ public function find($id = null)
 		return $row['data'];
 	}
 
+	/**
+	 * Fetches the column of database from $this->table with a primary key
+	 * matching $id.
+	 *
+	 * @param mixed|array|null $id One primary key or an array of primary keys
+	 * @param mixed|array|null $column_name Column name or null for primaryKey
+	 *
+	 * @return array|null array of table's column values
+	 */
+	public function findColumn($id = null, string $column_name = null) : array
+	{
+
+		if(is_null($column_name))
+			$column_name = $this->primaryKey;
+
+		if(!is_string($column_name))
+			throw DataException::forFindColumnIsNotAString();
+
+		$this->select($column_name);
+		$tmp = $this->asArray()->find($id);
```

Comment:
Why not chain these two statements together?

Reply:
don't know :). will change it.